examples/nidcpower_source_dc_voltage/measurement.py

```diff
@@ -68,7 +68,7 @@ def measure(
 
     session_management_client = create_session_management_client(measurement_service)
 
-    with session_management_client.reserve_sessions(
+    with session_management_client.reserve_session(
```

Comment:
FYI @jarnoldNI: this removes some more boilerplate code. If you only want one session, call reserve_session() instead of reserve_sessions(). It will validate that only one session is reserved, and `reservation.session_info` will be a single `SessionInformation` object instead of a list.